src/oas3/transformers/content.ts

```diff
@@ -108,17 +108,22 @@ export function translateHeaderObject(headerObject: unknown, name: string): Opti
 export function translateMediaTypeObject(mediaObject: unknown, mediaType: string): Optional<IMediaTypeContent> {
   if (!isDictionary(mediaObject)) return;
 
-  const { schema, example, examples, encoding } = mediaObject;
+  const { schema, encoding } = mediaObject;
+
+  const jsonSchema = schema
+    ? (toJsonSchema(schema, {
+        cloneSchema: true,
+        strictMode: false,
+        keepNotSupported: ['example', 'deprecated', 'readOnly', 'writeOnly', 'xml', 'externalDocs'],
+      }) as JSONSchema4)
+    : undefined;
+
+  const example = mediaObject.example || jsonSchema?.example;
+  const examples = mediaObject.examples || transformSchemaExamples(jsonSchema?.examples || jsonSchema?.['x-examples']);
```

Comment:
OAS3 Schema object has nor the examples nor the x-examples keywords
```suggestion
  const examples = mediaObject.examples;
```

Reply:
Right, not OAS3, it's JsonSchema keyword and the initial @marbemac idea was to allow usage of any of those example keywords in schema object, that's why I put it here. If we can't do that we would need to stick only with `example` in that case.